docs/versioned_docs/version-v3.2.0/02-Features/endpoint-validation.md

```diff
@@ -0,0 +1,215 @@
+﻿---
+title: "Endpoint validation"
+layout: default
+---
+
+# Endpoint validation
+
+We provide support for endpoint validation, when implementing your own custom web hook.
+
+- [Endpoint validation](#endpoint-validation)
+  - [Installation](#installation)
+  - [Azure Event Grid authorization](#azure-event-grid-authorization)
+    - [Enforce authorization globally](#enforce-authorization-globally)
+      - [Configuration](#configuration)
+    - [Enforce authorization per controller or operation](#enforce-authorization-per-controller-or-operation)
+      - [Configuration](#configuration-1)
+  - [Azure Event Grid subscription validation](#azure-event-grid-subscription-validation)
+    - [Enforce subscription validation per controller or operation](#enforce-subscription-validation-per-controller-or-operation)
+    - [Use subscription validation in Azure Functions](#use-subscription-validation-in-azure-functions)
+
+## Installation
+
+The features described here require the following package:
+
+> **⚠ This package used to be called `Arcus.EventGrid.Security`. Please make sure that you migrate towards this package as it's not being actively maintained beyond v3.2.**
```

Comment:
Instead of saying 'this package', I'd explicitly mention the name of the new package.
I'd also put this warning below the code-sample that shows to command to install the package

Reply:
Yes, will do! 👍  Thx!

Reply:
Done in https://github.com/arcus-azure/arcus.eventgrid/pull/232/commits/1a1df129b22cc0fe3d30d65319152f00a54e7ffe